dist/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+
+
+
+    <script id="entry-template" type="text/x-handlebars-template">
```

Comment:
This `script` should be in the `body` (not in the `head`). It can go right before all the imports.